.eslintrc.js

```diff
@@ -53,6 +53,9 @@ module.exports = {
     'no-magic-numbers': 'off',
     'sort-keys': 'off',
 
+    // Does not play well with react nodes
+    'no-param-reassign': [2, { props: false }],
```

Comment:
when assign properties to react nodes eslint throws an error, lets make prop assignment ok.